Library/Homebrew/os.rb

```diff
@@ -1,6 +1,6 @@
 module OS
   def self.mac?
-    /darwin/i === RUBY_PLATFORM
+    /darwin/i === RUBY_PLATFORM || ENV["HOMEBREW_EMULATE_MAC"]
```

Comment:
I just realize it's broken now. I will add on how to at least make `brew info --json=v1` workable on Linux to my TODO list.